src/core/component/directives/resize/interface.ts

```diff
@@ -28,12 +28,11 @@ export interface ObserverOptions {
 	watchHeight?: boolean;
 
 	/**
-	 * Callback that will be called if the element size has been changed
-	 *
-	 * @param observable
-	 * @param oldSize
-	 * @param newSize
+	 * If true, the callback will be called immediately after the installation of the directive
```

Comment:
If true, then the callback is invoked immediatelly after initializing of the directive